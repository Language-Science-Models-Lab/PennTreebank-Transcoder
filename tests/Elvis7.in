
(S @(NP @(NNP @Elvis)) (VP (NP @(NNP @Elvis)) (S (NP @(NNP @Elvis)) @(VP @(VBZ @is) (ADJP-PRD~ @(JJ @dead))))) (. @.))

# tests phrase rules 7

# note the test sentences are not necessarily grammatical English and would not be found in the actual Treebank 

