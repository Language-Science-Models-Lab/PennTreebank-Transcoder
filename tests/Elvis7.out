[is dead]:type(type(type(is, _)@type(dead, _), _), s)
